cfgov/unprocessed/css/organisms/filterable-list-controls.less

```diff
@@ -0,0 +1,40 @@
+/* topdoc
+  name: Filterable List Controls
+  family: cf-organisms
+  patterns:
+  - name: Filterable List Controls
+    markup: |
+      <div class="o-filterable-list-controls">
+          <div class="m-expandable
+                      m-expandable__borders
+                      m-expandable__midtone
+                      m-expandable__expanded">
+              …
+          </div>
+          <div class="m-notification"></div>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        .o-filterable-list-controls
+          .m-expandable
+            [...]
+          .m-notification
+            [...]
+  tags:
+  - cf-organisms
+*/
+
+.o-filterable-list-controls {
+    // Add gap between expandable and notification.
+    .m-notification {
+        margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
```

Comment:
So `block` would work when the notification is showing, but it won't when the notification is hidden because it'll still apply the margin (padding in the case of `block`) and create a gap below the filter.